crates/bevy_macro_utils/src/lib.rs

```diff
@@ -44,11 +44,17 @@ impl BevyManifest {
         };
 
         let deps = self.manifest.dependencies.as_ref();
-        let deps_dev = self.manifest.dev_dependencies.as_ref();
 
-        deps.and_then(find_in_deps)
-            .or_else(|| deps_dev.and_then(find_in_deps))
-            .unwrap_or_else(|| get_path(name))
+        let path = deps.and_then(find_in_deps);
```

Comment:
```suggestion
        #[cfg(not(test))]
        let path = deps.and_then(find_in_deps);
```

Reply:
or alternatively do 
```rust
let path = if cfg!(test) {
    //...
} else {
    // ...
}
```

Reply:
I actually had something like this originally but s33n on Discord pointed out that `[dev-dependencies]` are additive, so if it exists in `[dependencies]` it should be included either way.

Reply:
ohh I see the log now.